linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/rule.yml

```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: ubuntu2004
+
+title: 'AIDE must notify personnel if baseline configurations are altered'
+
+description: |-
+    The operating system file integrity tool must be configured to notify designated personnel of any changes to configurations.
+
+rationale: |-
+    Detecting changes in the system can help avoid unintended, and negative consequences 
+    that could affect the security state of the operating system
+
+severity: medium
+
+references:
+    disa: 'CCI-001744,CCI-002702'
+    srg: 'SRG-OS-000447-GPOS-00201,SRG-OS-000363-GPOS-00150'
+    stigid@ubuntu2004: UBTU-20-010437
+
+ocil_clause: 'silentreports is enabled in aide default configuration, or is missing'
+
+ocil: |- 
+    Verify that Advanced Intrusion Detection Environment (AIDE) notifies the System Administrator
+    when anomalies in the operation of any security functions are discovered with the following command: 
+    <pre># grep SILENTREPORTS /etc/default/aide </pre>
+
+    SILENTREPORTS=no 
+
+    If SILENTREPORTS is commented out, this is a finding. 
+ 
+    If SILENTREPORTS is set to "yes", this is a finding. 
+ 
+    If SILENTREPORTS is not set to "no", this is a finding.
+
+fixtext: |-
+    Configure the Ubuntu operating system to notify designated personnel if baseline configurations are changed in an unauthorized manner. 
```

Comment:
replace Ubuntu with {{{ full_name }}}

Reply:
Replaced with `{{ full_name }}` in https://github.com/ComplianceAsCode/content/pull/10738/commits/c7c9463317a6570a08df6710bd1d98692ea3ae11